base/i18n/icu_util.cc

```diff
@@ -86,6 +86,11 @@ void LazyInitIcuDataFile() {
   if (g_icudtl_pf != kInvalidPlatformFile) {
     return;
   }
+//It's not correct to check and read from assets folder first, since xwalk
+//put icudtl.dat in other folders like res/raw instead of assets.Reading
+//from assets has a risk of conflict with android system webview.
+//See https://crosswalk-project.org/jira/browse/XWALK-7004
+#if 0
 #if defined(OS_ANDROID)
```

Comment:
Is it code style requirement?
I don't think it's a good practice to replace #if(OS_ANDROID） with #if 0.
Why not remove them in such case?